core/src/yamlscript/resolver.clj

```diff
@@ -66,13 +67,20 @@
 ;; ----------------------------------------------------------------------------
 ;; Resolve taggers for code mode:
 ;; ----------------------------------------------------------------------------
+(defn tag-str [[key val]]
+  (when-let*
+    [str (:ysi key)
+     _ (= "" (:ysx val))]
+    [{:str str} {:str ""}]
```

Comment:
reformat this (blank line, remove the spacing before the `))` next line

Reply:
fixed